src/lammps.cpp

```diff
@@ -48,6 +48,7 @@
 #include "version.h"
 #include "memory.h"
 #include "error.h"
+#include "mdi_init.h"
```

Comment:
You must not unconditionally include a header file from a package.
This will break compilation for all situations where the package is not installed.